aten/src/ATen/native/cuda/linalg/BatchLinearAlgebraLib.cpp

```diff
@@ -93,13 +86,13 @@ void geqrf_batched_cublas(const Tensor& input, const Tensor& tau) {
 
 template <typename scalar_t>
 static void apply_lu_solve_batched_cublas(const Tensor& b, const Tensor& lu, const Tensor& pivots, TransposeType transpose) {
-#ifndef CUDART_VERSION
-  TORCH_CHECK(false, "lu_solve: cuBLAS backend for lu_solve is not available.")
-#else
   TORCH_INTERNAL_ASSERT(batchCount(b) == batchCount(lu), "batch_size of b and lu must be the same");
   TORCH_INTERNAL_ASSERT(batchCount(lu) == batchCount(pivots.unsqueeze(-1)), "batch_size of lu and pivots must be the same");
+#ifdef ROCM_VERSION
```

Comment:
Did you mean to use ROCM_VERSION instead of USE_ROCM?
```suggestion
#ifdef USE_ROCM
```